### ISSUE TITLE
Do Perplexity

### ISSUE THREAD

**Issue by rudyrdx** [rahulsushilsharma/huggingface-chat#33]
**Is your feature request related to a problem? Please describe.**
Make one for perplexity

**Describe the solution you'd like**
They are fast

**Describe alternatives you've considered**
chat.lmsys.org

**Additional context**
collab?

**Comment by rahulsushilsharma** [comment]
that is a nice idea, will do some research

**Comment by rudyrdx** [comment]
@rahulsushilsharma any updates ?

**Comment by rahulsushilsharma** [comment]
yup checked the api's the implimentations won't require a auth and not a hugging-face api's so maybe will create a seprate package for that

**Comment by rudyrdx** [comment]
ok so the above is i am assuming for perplexity right ? sure perplexity is fast n all but quality lies in chat.lmsys.org, i have done perplexity, if any doubts feel free to ask. but unable to find time for chat.lmsys.org.

**Comment by rahulsushilsharma** [comment]
yea I also noticed  chat.lmsys.org is quite good but I am not sure about the limits and also the api workinngs are quite weird? I would love to start a project for that so if you have any research done on it please share it will help. 

and also can I check out your project on perplexity?

**Comment by rudyrdx** [comment]
sure, Ill add you as a collab

**Comment by rudyrdx** [comment]
Did you try the perplexity wrapper ?

**Comment by rahulsushilsharma** [comment]
sorry, didn't get time to check will do now

**Comment by rahulsushilsharma** [comment]
where will i get the key? do I have to sub?

**Comment by rudyrdx** [comment]
no no
just run the Soc.py
or better do you have discord ?
my id : rudyrdx

**Comment by rudyrdx** [comment]
any updates ?

**Comment by rahulsushilsharma** [comment]
hi sorry am little sick rn won't be able to work for sometime.

**Comment by rudyrdx** [comment]
kein problem take care

**Comment by rudyrdx** [comment]
hi, any updates?

**Comment by rahulsushilsharma** [comment]
Hi, yea no sorry go RSI symptoms, ugh...
